Tensile/KernelWriterAssembly.py

```diff
@@ -3186,13 +3187,15 @@ def functionSignature(self, kernel ):
           msg = "reading and writing LDS at same time require 2 LDS buffer"
       elif self.overflowedResources == 6:
         msg = "SIA2 better with occupancy 2"
+      elif self.overflowedResources == 7:
+        msg = "too many accvgprs"
       else:
         msg = "unknown"
 
       if globalParameters["PrintSolutionRejectionReason"]:
-        printWarning("%s overflowed resources.  errorCode=%d, msg=\"%s\", vgprs=%u, sgprs=%u" \
+        printWarning("%s overflowed resources.  errorCode=%d, msg=\"%s\", vgprs=%u, sgprs=%u, accvgprs=%u" \
```

Comment:
I am thinking it might be better to print accvgprs only when kernel["EnableMatrixInstruction"] is true.

Reply:
This will not cause an error even if kernel["EnableMatrixInstruction"] is false, but it is unnecessary information.